docs/data/external-oasis-repos.md

```diff
@@ -29,26 +29,26 @@ Requirements: setting up a CVMFS repository server
 
 This document doesn't cover requirements for the GOC computers since those are already set up. The requirements for a CVMFS repository server are that it have installed cvmfs-server and cvmfs client, both version 2.2.2 or greater; version 2.3.3 or greater is recommended on EL6 and required on EL7. This requires using aufs or OverlayFS: aufs requires a modified kernel from CERN for Redhat EL6-based systems, and OverlayFS requires at least EL7.3; the latter uses a standard kernel so it is highly recommended. Also an apache httpd server needs to be enabled. NOTE: multiple cvmfs repositories can be hosted on the same machine.
 
-`/srv/cvmfs` will hold all the published data, so make sure it is large enough to hold all the repositories hosted on the machine. `/var/spool/cvmfs` will hold files during publish, so it should be large enough to hold the most amount of data that might be attempted to be published at once. In addition, on EL7.3 and cvmfs-server-2.3.3 or 2.3.5, `/var/spool/cvmfs` needs to be of filesystem type 'ext4' or the `cvmfs_server` command will not allow it. There is a variable to override the filesystem type check, but other filesystem types are not guaranteed to work (ext3 and xfs created with `ftype=1` should work with `CVMFS_DONT_CHECK_OVERLAYFS_VERSION=yes`).
+`/srv/cvmfs` will hold all the published data, so make sure it is large enough to hold all the repositories hosted on the machine. `/var/spool/cvmfs` will hold files during publish, so it should be large enough to hold the most amount of data that might be attempted to be published at once. In addition, on EL7 and cvmfs-server-2.4.1, `/var/spool/cvmfs` needs to be of filesystem type 'ext3' or 'ext4', or the `cvmfs_server` command will not allow it. There is a variable to override the filesystem type check, but other filesystem types are not guaranteed to work (xfs created with `ftype=1` should work with `CVMFS_DONT_CHECK_OVERLAYFS_VERSION=yes`).
 
 This is a procedure for installing it on a Redhat EL6-based system:
 
 ``` console
 root@host # rpm -i https://cvmrepo.web.cern.ch/cvmrepo/yum/cvmfs-release-latest.noarch.rpm 
-root@host # rpm -i https://repo.grid.iu.edu/osg/3.3/osg-3.3-el6-release-latest.rpm 
+root@host # rpm -i https://repo.grid.iu.edu/osg/3.4/osg-3.4-el6-release-latest.rpm 
 root@host # yum install --enablerepo=cernvm-kernel --disablerepo=cernvm kernel aufs2-util cvmfs-server.x86_64 cvmfs.x86_64 cvmfs-config-osg 
 root@host # echo "cvmfs_server mount -a" >>/etc/rc.local 
 root@host # reboot</pre>
 ```
-
-This is the procedure for installing on a Redhat EL7.3-based system:
+<br/>
+This is the procedure for installing on a Redhat EL7-based system:
 
 ``` console
-root@host # rpm -i https://repo.grid.iu.edu/osg/3.3/osg-3.3-el7-release-latest.rpm 
+root@host # rpm -i https://repo.grid.iu.edu/osg/3.4/osg-3.4-el7-release-latest.rpm 
 root@host # yum install cvmfs-server.x86_64 osg-oasis 
 root@host # echo "cvmfs_server mount -a" >>/etc/rc.local</pre>
```

Comment:
Seems that touching `/etc/rc.local` is bad in all cases that I know about... this should be a proper system service or not advised at all.

Reply:
Why is it bad?  What problems does it cause?  The upstream cvmfs documentation suggests to put it there.

Reply:
`/etc/rc.local` is problematic because:
- It's unclear what's prior in the script, or if the script is executed in `-e` mode -- meaning this might never actually run!
- There's no resource management, dependency management, or service lifetime management as is done for systemd.
- There's no parallelism.  Everything stops and waits on CVMFS.
- The RHEL7 default script includes: `It is highly advisable to create own systemd services or udev rules to run scripts during boot instead of using this file.`

It's just not the proper way of doing things anymore... we should contact upstream CVMFS about this advice too.

Fix it if it's easy, but I'm not going to hold up the PR for it.

Reply:
Making a systemd service is so much more complicated so I'd rather avoid that ... I did however find by reading the /etc/rc.local comments that in order to enable it you have to do chmod +x /etc/rc.local, so I added that.  I verified that the rc-local systemd service runs after network, and that's good enough for this.